From 5fcf8f8b67d298a8379c9039ea75d0f98b915422 Mon Sep 17 00:00:00 2001
From: Thomas Ward <teward@thomas-ward.net>
Date: Sun, 7 Aug 2022 17:14:36 -0400
Subject: [PATCH] Fix python strict type checking in Python 3.9+

Python 3.9+ started doing strict enforcement of types.  This results in the GUI function methods not functioning properly in modern Python environments.

Fixes #628

Thanks to @icceland in bug 628 for the fix stated in the bug.
---
 torbrowser_launcher/__init__.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/torbrowser_launcher/__init__.py b/torbrowser_launcher/__init__.py
index bff3e8e..115fd4f 100644
--- a/torbrowser_launcher/__init__.py
+++ b/torbrowser_launcher/__init__.py
@@ -90,8 +90,8 @@ def main():
     desktop = app.desktop()
     window_size = gui.size()
     gui.move(
-        (desktop.width() - window_size.width()) / 2,
-        (desktop.height() - window_size.height()) / 2,
+        (desktop.width() - window_size.width()) // 2,
+        (desktop.height() - window_size.height()) // 2,
     )
     gui.show()
 
